browser-test/src/admin/admin_bulk_update_statuses.test.ts

```diff
@@ -0,0 +1,220 @@
+import {test, expect} from '../support/civiform_fixtures'
+import {Page} from 'playwright'
+import {
+  loginAsAdmin,
+  loginAsProgramAdmin,
+  loginAsTestUser,
+  logout,
+  enableFeatureFlag,
+  waitForPageJsLoad,
+  testUserDisplayName,
+  supportsEmailInspection,
+  extractEmailsForRecipient,
+  AdminPrograms,
+  ApplicantQuestions,
+  AdminProgramStatuses,
+} from '../support'
+
+test.describe('with program statuses', () => {
+  const programName = 'Applicant with statuses program'
+  const approvedStatusName = 'Approved'
+  const rejectedStatusName = 'Rejected'
+  const reapplyStatusName = 'Reapply'
+
+  test.beforeEach(
+    async ({page, adminPrograms, adminProgramStatuses, applicantQuestions}) => {
+      await enableFeatureFlag(page, 'bulk_status_update_enabled')
+      await loginAsAdmin(page)
+
+      await adminPrograms.addProgram(programName)
+      await adminPrograms.gotoDraftProgramManageStatusesPage(programName)
+      await adminProgramStatuses.createStatus(approvedStatusName)
+      await adminProgramStatuses.createStatus(rejectedStatusName)
+      await adminProgramStatuses.createStatus(reapplyStatusName)
+      await adminPrograms.publishProgram(programName)
+      await adminPrograms.expectActiveProgram(programName)
+      await logout(page)
+
+      // Submit an application as a test user so that we can navigate back to the applications page.
+      await loginAsTestUser(page)
+      await applicantQuestions.clickApplyProgramButton(programName)
+      await applicantQuestions.submitFromReviewPage()
+      await logout(page)
+      await loginAsProgramAdmin(page)
+      await adminPrograms.viewApplications(programName)
+    },
+  )
+
+  test('single application updated with bulk status select', async ({
+    page,
+    adminPrograms,
+  }) => {
+    // Default page shows all applications.
+    await adminPrograms.expectApplicationCountForBulkStatus(1)
+    await adminPrograms.expectApplicationHasStatusStringForBulkStatus(
+      testUserDisplayName(),
+      'None',
+    )
+    await page.locator('#selectAll').check()
+    await page.locator('#bulk-status-selector').selectOption('Approved')
```

Comment:
(note) Typically I would push to not reference things by ID. However given this is the admin side and doesn't have the accessibility or labels needed. And given that this UI will likely be getting a bit of an overhaul when the design team can get a fuller feature design, this is fine for now.

Reply:
Agree! Am I good to merge @gwendolyngoetz ?